docs/tutorials/hostpath/xfs_quota/use-xfs-fs-with-loop-device.md

```diff
@@ -0,0 +1,34 @@
+# Create XFS filesystem at the basepath as loop device (if filesystem is not XFS)
```

Comment:
Similarly we can add link to this doc specifying how to bring up a test environment to make use of the quota feature?

Reply:
Added a note section with a link to the loop device setup.